### PR TITLE
Clean dependencies

### PR DESCRIPTION
**Pull request by orblazer**
Hello,
I have clean unused nd deprecated dependencies. Clean PR from #1125 

The packages `clean-css`, `less` and `source-map` is required in `package.json` ?

Thanks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,19 +99,8 @@
     "logo": "https://opencollective.com/opencollective/logo.txt"
   },
   "config": {
-    "validate-commit-msg": {
```
Comment (nchanged) [inline package.json]: Why has it been removed? scope is optional in our case

Comment (orblazer) [reply package.json]: I have replace deprecated `validate-commit-msg` by `@commitlint/cli` and `@commitlint/config-conventional`.

Comment (nchanged) [reply package.json]: Make sure that commits like: `chore: foo bar` yields at all. We have scope as an optional parameter

**Comment by jpike88** [comment]
@orblazer what the status on this one? just trying to help clean things up

**Comment by jpike88** [comment]
@orblazer knock knock...

**Comment by orblazer** [comment]
Hello @jpike88 , i don't have pick a time to fix conflict fpr merging and this PR is just for clean depencies

**Comment by Kompwu** [comment]
@everyone
@nchanged said he had to test before.
Because he doesn't have the time, if someone could test it out it would be great ! :)
If nobody have the time to test it, i'll do it ;)

**Comment by orblazer** [comment]
@Kompwu i have very fast try but currently i search how i can fix the errors on vetur for VS Code

**Comment by Kompwu** [comment]
@orblazer np notify me when conflicts are resolved and i'll do it :wink:

**Comment by Kompwu** [comment]
@orblazer Knock knock ?

**Comment by orblazer** [comment]
ok sry @Kompwu for the late :)

**Comment by Kompwu** [comment]
@orblazer Everything works :ok_hand: 
BTW, There are still some dependencies depreciated:
```
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9a
c5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated natives@1.1.3: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4
.x.
npm WARN deprecated gulp-util@2.2.20: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9
ac5
```
Can you solve these ?
@nchanged Knock knock... safe to merge :grinning:

**Comment by orblazer** [comment]
@Kompwu hello, i can't fix that because is it caused by depencies and i have all latest version

**Comment by nchanged** [comment]
@orblazer how sure you are that it won't introduce any breaking changes?

**Comment by orblazer** [comment]
@nchanged @Kompwu is supossed testing if don't break any changes

**Comment by Kompwu** [comment]
@nchanged Anyway, I couldn't find anything broken & Travis CI build passed.
The best thing is that you also check to be sure.

**Comment by Kompwu** [comment]
@nchanged How's going ?

File: package.json (hunk 1 above)
Comment (nchanged) [reply package.json]: we need `scope require false` option.

**Comment by Kompwu** [comment]
@orblazer Knock knock ...

**Comment by Kompwu** [comment]
@nchanged Give me WR on this PR, i'm gonna fix it.

**Comment by Kompwu** [comment]
@orblazer Hey mate?

**Comment by orblazer** [comment]
@Kompwu sorry for my AFK im working on other project.

I have look de review but i dont know the value need add in `commitlintrc` i have found only `never` and `always` parameters.
If you can fix that i am really happy

**Comment by Kompwu** [comment]
@orblazer `never` is the option we're looking for.

**Comment by orblazer** [comment]
@Kompwu `never` lock commit with scope

**Comment by Kompwu** [comment]
@orblazer hem, maybe try using `false`

**Comment by Kompwu** [comment]
@orblazer try `always` too.

File: package.json (hunk 1 above)
Comment (fmaita) [reply package.json]: Yes

Comment (Kompwu) [reply package.json]: It's already optional since we're using `@commitlint/config-conventional`.

File: package.json [hunk 2]
```diff
@@ -79,8 +79,11 @@
     "fuse-concat-with-sourcemaps": "^1.0.5",
     "glob": "^7.1.2",
     "ieee754": "^1.1.11",
+    "inquirer": "^3.0.6",
     "lego-api": "^1.0.8",
+    "minimist": "^1.2.0",
     "mustache": "^2.3.0",
+    "node-sass": "^4.9.0",
     "postcss": "^6.0.22",
```
Comment (nchanged) [inline package.json]: Node sass might have slipped by mistake

Comment (Kompwu) [reply package.json]: @nchanged wtf?
@orblazer Please.... squash your commits & follow `git-cz` rules for scope, etc...

Comment (orblazer) [reply package.json]: Yes I have resolve the conflit from my phone and on github we can't resolve conflit with custom commit message :(

Comment (Kompwu) [reply package.json]: @nchanged What you mean by "Node sass might have slipped by mistake" ?

Comment (Kompwu) [reply package.json]: @orblazer so?

File: package.json [hunk 3]
```diff
@@ -33,95 +33,77 @@
...
-    "glob": "^7.1.1",
-    "ieee754": "^1.1.8",
+    "glob": "^7.1.2",
+    "ieee754": "^1.1.11",
     "inquirer": "^3.0.6",
-    "lego-api": "^1.0.7",
+    "lego-api": "^1.0.8",
     "minimist": "^1.2.0",
     "mustache": "^2.3.0",
     "node-sass": "^4.9.0",
```
Comment (Kompwu) [inline package.json]: Please remove `node-sass`.

**Comment by Kompwu** [comment]
@orblazer Please close this PR.
#1331 replace it.

**Pull request closed without merging**